tests/httpwrappers/tests.py

```diff
@@ -317,14 +317,14 @@ def test_headers_type(self):
         with self.assertRaises(UnicodeError):
             r.__setitem__('føø', 'bar')
         with self.assertRaises(UnicodeError):
-            r.__setitem__('føø'.encode('utf-8'), 'bar')
+            r.__setitem__('føø'.encode(), 'bar')
 
     def test_long_line(self):
         # Bug #20889: long lines trigger newlines to be added to headers
         # (which is not allowed due to bug #10188)
         h = HttpResponse()
         f = 'zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz a\xcc\x88'.encode('latin-1')
-        f = f.decode('utf-8')
+        f = f.decode()
```

Comment:
And once more :). These tests that deliberately mix encodings seem more obvious with the encoding explicitly stated